_data/simple-icons.json

```
@@ -9020,6 +9020,12 @@
             "hex": "83DA77",
             "source": "https://www.paloalto.com"
         },
+        {
+            "title": "Panasonic",
+            "hex": "0049AB",
+            "source": "https://www.panasonic.com/content/dam/Panasonic/plogo.svg",
```

```suggestion
            "source": "https://www.panasonic.com",
```
Although we're using a specific SVG, it's available in the website header. No need to link directly to the file.